### PR TITLE
Add eu.scarpetta.PDFMixTool

### PR DESCRIPTION
**Pull request by marcoscarpetta**

### CODE REVIEW

File: eu.scarpetta.PDFMixTool.yaml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+runtime: org.kde.Platform
+runtime-version: 5.11
+sdk: org.kde.Sdk
+command: pdfmixtool
+finish-args:
+- --share=ipc
+- --socket=x11
+- --socket=wayland
+- --device=dri
+- --filesystem=home:create
```
Comment (barthalion) [inline eu.scarpetta.PDFMixTool.yaml]: `:create` makes no sense in this context; have you checked if it works properly with desktop portal so it doesn't need legacy permissions like this?

File: eu.scarpetta.PDFMixTool.yaml [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+modules:
+- name: podofo
+  buildsystem: cmake
```
Comment (barthalion) [inline eu.scarpetta.PDFMixTool.yaml]: cmake-ninja

File: eu.scarpetta.PDFMixTool.yaml [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+  sources:
+  - type: archive
+    url: https://datapacket.dl.sourceforge.net/project/podofo/podofo/0.9.6/podofo-0.9.6-rc1.tar.gz
+    sha256: ce7a8acd7e127c9529bbf26f174583177b70e100aa356afc323c7677a3827c92
+  cleanup:
+  - /include
+  - /lib/pkgconfig
+  - /lib/debug/source
+- name: pdfmixtool
+  buildsystem: cmake
```
Comment (barthalion) [inline eu.scarpetta.PDFMixTool.yaml]: cmake-ninja

File: eu.scarpetta.PDFMixTool.yaml [hunk 4]
```diff
@@ -0,0 +1,37 @@
+app-id: eu.scarpetta.PDFMixTool
```
Comment (barthalion) [inline eu.scarpetta.PDFMixTool.yaml]: Please follow our YAML style guide: https://github.com/flathub/flathub/wiki/YAML-Style-Guide

**Comment by barthalion** [comment]
bot, build  eu.scarpetta.PDFMixTool

**Comment by nedrichards** [comment]
bot, build eu.scarpetta.PDFMixTool

**Comment by barthalion** [comment]
Repository has been created: https://github.com/flathub/eu.scarpetta.PDFMixTool

Make sure to accept the invitation to collaborators to get the write access to the repo. Thanks!

**Pull request closed without merging**